private-pools-example/README.md

```diff
@@ -0,0 +1,294 @@
+
+In line 22 of `cloudbuild.yaml`, replace `<add-ip-address>` with the IP address for our private application on GKE, which we can access by running `echo $ILB`.
+
+```
+  args: ['<add-ip-address>:80']
+```
+
+In line 6, replace `<add-project-id>` with your GCP Project ID, `<add-region>` with the value stored in `$REGION`, and `<add-privatepools-name>` with the value stored in `$PRIVATE_POOL_NAME`.
+
+```
+  workerPool: 'projects/<add-project-id>/locations/<add-region>/workerPools/<add-privatepools-name>'
+```
+
+Now we can submit our build!
+```
+gcloud builds submit --config=cloudbuild.yaml
+```
+
+Your initial output will look similar to the below.
+```
+Creating temporary tarball archive of 4 file(s) totalling 997 bytes before compression.
+Uploading tarball of [.] to [gs://agmsb-k8s_cloudbuild/source/1627484214.946535-9f309a0eec9c4296a6d76d032cb87ebe.tgz]
+Created [https://cloudbuild.googleapis.com/v1/projects/agmsb-k8s/locations/us-west1/builds/07c6bbb5-5044-47f8-b508-75758cb7bd61].
+Logs are available at [https://console.cloud.google.com/cloud-build/builds;region=us-west1/07c6bbb5-5044-47f8-b508-75758cb7bd61?project=381015962062].
+```
+
+Feel free to click the second hyperlink to access logs and details about the build. 
+
+The build output, once completed, should look like the below:
+
+```
+
+----------------------------------------------------------- REMOTE BUILD OUTPUT -----------------------------------------------------------
+starting build "07c6bbb5-5044-47f8-b508-75758cb7bd61"
+
+FETCHSOURCE
+Fetching storage object: gs://agmsb-k8s_cloudbuild/source/1627484214.946535-9f309a0eec9c4296a6d76d032cb87ebe.tgz#1627484215665319
+Copying gs://agmsb-k8s_cloudbuild/source/1627484214.946535-9f309a0eec9c4296a6d76d032cb87ebe.tgz#1627484215665319...
+/ [1 files][  857.0 B/  857.0 B]                                                
+Operation completed over 1 objects/857.0 B.
+BUILD
+Already have image (with digest): gcr.io/cloud-builders/curl
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100    68  100    68    0     0  12495      0 --:--:-- --:--:-- --:--:-- 13600
+Hello, world!
+Version: 2.0.0
+Hostname: private-app-65fd4cd975-m4gm2
+PUSH
+DONE
+-------------------------------------------------------------------------------------------------------------------------------------------
+```
+
+In the build logs, we can see that the `curl` build step completed -- were able to successfully access a private ILB from a build running on Cloud Build private pools!
+
+## Cleanup
+
+Delete the GKE cluster.
+```
+gcloud container clusters delete $GKE_CLUSTER_NAME --region $REGION
+```
+
+Delete the Cloud Build Private Pool. 
+```
+gcloud alpha builds worker-pools delete $PRIVATE_POOL_NAME --region $REGION
```

Comment:
Can we remove this alpha reference?

Reply:
Updated in https://github.com/GoogleCloudPlatform/cloud-build-samples/pull/13/commits/5b8ea2b1e4beacca4edf5eea39f119d396016fcf